backend/src/models/likes.js

```diff
@@ -34,13 +34,46 @@ class Likes {
      * Put likes
      */
     putLikes({ type, id, userID, value }) {
-        return this.db
-            .insertUnique('likes', {
-                objectType: type,
-                objectID: id,
-                userID,
-                value
-            })
+
+        // Stuff to update likes
+        const data = {
+            objectType: type,
+            objectID: id,
+            userID,
+            value
+        }
+        const insertValues = Object.values(data)
+        const insertColumns = Object.keys(data)
+        const insertPlaceholders = insertColumns.map(_ => '?').join()
+        const updateLikes = `REPLACE INTO likes (${insertColumns}) VALUES (${insertPlaceholders})`
+
+        // Stuff to update reputation
+        const creatorTable = (type === 'question' || type === 'review') ? type : 'comment'
+        const updateReputation = `UPDATE user
+        SET reputation = (SELECT reputation FROM user WHERE id = ?) + ?
```

Comment:
pretty sure this can just be `SET reputation = reputation + ?`

Reply:
Doesn't seem to work

Reply:
weird... it works for the SET in course.js